tests/nat/middleware/test_defense_middleware_pii.py

```
@@ -0,0 +1,666 @@
+
+        async def mock_next(_value):
+            return {
+                "items": [
+                    {"email": "first@example.com", "id": 1},
+                    {"email": "second@example.com", "id": 2},
+                    {"email": "third@example.com", "id": 3}
+                ]
+            }
+
+        with patch('nat.middleware.defense_middleware_pii.logger'):
+            result = await middleware.function_middleware_invoke({}, mock_next, middleware_context)
```

Ruff (0.14.8)

Local variable `result` is assigned to but never used

Remove assignment to unused variable `result`

(F841)